docs/source/onnxruntime/modeling_ort.mdx

```diff
@@ -0,0 +1,112 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Optimum Inference with Onnxrunime
+
+Optimum Inference is a utility package for building and running inference with accleratored runtime like Onnxruntime. 
+Optimum Inference can be used to load optimized models from the [Hugging Face Hub](hf.co/models) and create pipelines 
+to run acclerated inference without rewriting your APIs.
+
+## Switching from Transfromers to Optimum Inference
+
+The Optimum Inference models are API compatible with Transformers models, meaning you can just replace your `AutoFor*` with the optimum `OnnxFor*` class. 
+
+```diff
+from transformers import AutoTokenizer, pipeline
+-from transformers import AutoModelForQuestionAnswering
++from optimum.onnxruntime import OnnxForQuestionAnswering
+
+-model = AutoModelForQuestionAnswering.from_pretrained("deepset/roberta-base-squad2")
++model = OnnxForQuestionAnswering.from_transformers("deepset/roberta-base-squad2")
+tokenizer = AutoTokenizer.from_pretrained("deepset/roberta-base-squad2")
+
+onnx_qa = pipeline("question-answering",model=model,tokenizer=tokenizer)
+
+question = "Whats my name?"
+context = "My Name is Philipp and I live in Nuremberg."
+pred = onnx_qa(question, context)
+```
+
+Optimum Inference also includes methods to convert vanilla Transformers models to optimized models (`.from_transfromers`). 
+After you have converted a model you can even `optimize` or `quantize` the model if it is supported by the runtime you use.
+
+```python
+from transformers import AutoTokenizer, pipeline
```

Comment:
We should use `>>>` for each line of code in the guides. This will allow `doc-builder` to generate Jupyter notebooks automatically for us